dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -277,21 +272,11 @@ private Set<TrackedEntityAttributeValue> getTrackedEntityAttributeValues(Enrollm
   }
 
   private List<Enrollment> findEnrollments(
```

Comment:
Should we rename this method now as it is only a mapping and no more filtering is happening?